target_chains/solana/sdk/js/pyth_solana_receiver/src/vaa.ts

```diff
@@ -97,7 +98,7 @@ export async function buildEncodedVaaCreateInstruction(
  * This number was chosen as the biggest number such that one can still call `createInstruction`, `initEncodedVaa` and `writeEncodedVaa` in a single Solana transaction.
  * This way, the packing of the instructions to post an encoded vaa is more efficient.
  */
-export const VAA_SPLIT_INDEX = 792;
+export const VAA_SPLIT_INDEX = 755;
```

Comment:
Need to be less aggressive here because Jito tip is now with the first write encoded vaa